File: TagMemo-java/app/src/main/java/com/timejh/tagmemo_java/model/GroupMemo.java
package com.timejh.tagmemo_java.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by tokijh on 2017. 4. 30..
 */

public class GroupMemo extends RealmObject {

    public static final int TYPE_GROUP = 0;
    public static final int TYPE_MEMO = 1;

    public String id;

    public int type;
    public String parentGroupId;
    public String type_id; // TYPE에 맞는 ID(Group 또는 Memo)
    public Group group;
    public Memo memo;

    public String last_date;
    public boolean isValidated;
}

File: TabMemo-Swift/TabMemo-Swift/GroupMemo.swift
//
//  GroupMemo.swift
//  TabMemo-Swift
//
//  Created by 윤중현 on 2017. 6. 3..
//  Copyright © 2017년 윤중현. All rights reserved.
//

import RealmSwift

class GroupMemo: Object {
    public static let type_group: Int = 1
    public static let type_memo: Int = 2
    
    dynamic var id: String = ""
    
    dynamic var type: Int = 0
    dynamic var parentGroupId: String = ""
    dynamic var type_id: String = "" // TYPE에 맞는 ID(Group 또는 Memo)
    dynamic var group: Group?
    dynamic var memo: Memo?
    
    dynamic var last_date: String = ""
    dynamic var isValidated: Bool = false
}
